motion_computation/docs/motion_computation_node.md

```diff
@@ -0,0 +1,74 @@
+# Motion computation Node
```

Comment:
This is cool, how are these generated?

Reply:
By hand. I modeled it off of autoware.univere's documentation. I'm not sure if they have an auto-generator for these types of file. I don't think they do since it varies from package to package.

I think this style of documentation should be standard in all our packages because it provides users a quick way to see a Node's interface.

Reply:
Yeah it'd be nice to have it for all the packages. We just have to be diligent in updating the documents if anything chagnes. Can we have this or the package_design.md as the README.md then as it looks much better?

Reply:
Sure thing!